src/markdown.c

```diff
@@ -1413,6 +1413,25 @@ prefix_quote(uint8_t *data, size_t size)
 	return 0;
 }
 
+static size_t
+prefix_blockspoiler(uint8_t *data, size_t size)
```

Comment:
(The only way it could hurt I guess is if we're wrong, this is a bit of spelunking for both of us I think.)